bokeh/settings.py

```diff
@@ -8,6 +8,13 @@
 ''' Control global configuration options with environment variables.
 A global settings object that other parts of Bokeh can refer to.
 
+``BOKEH_ALLOW_WS_ORIGIN`` --- List of websocket origins allowed to access bokeh.
+
+  Comma separated list of domains that need to access bokeh websocket interface.
+  This can also be provided using the --allow-websocket-origin parameter. 
+
+  Note: No option overrides the other. We instead merge the list from --allow-websocket-origin
+  and the list from BOKEH_ALLOW_WS_ORIGIN
```

Comment:
I think this merits some discussion. My experience is that "standard" semantics for this sort of thing is for the env var to completely override other settings. I think merging might be surprising to users.